pyaedt/edb_core/layout_validation.py

```diff
@@ -230,3 +230,21 @@ def illegal_net_names(self, fix=False):
 
         self._pedb._logger.info("Found {} illegal net names.".format(len(renamed_nets)))
         return
+
+    def illegal_rlc_values(self, fix=False):
+        """Find and fix rlc illegal values."""
```

Comment:
```suggestion
        """Find and fix RLC illegal values."""
```
Based on a search of the doc, it looks like we use RLC in all capital letters in descriptions.